releasenotes/notes/python-3-8-c66cde69f972eb1e.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note is combined when the
+# CHANGELOG.rst is rendered. Therefore, the text should be worded in a way
+# that does not depend on any information available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    Upgrade embedded Python 3 to 3.8.1
```

Comment:
It's the first time we upgrade the embedded Python 3 version, so I think it's worth being a bit explicit about the impact:
* mention explicitly here that v6 Agents continue to ship python 2 as well, at version `2.7.17` (unchanged)
* nit: since we upgrade to a new minor version of python (as opposed to a patch version), I think it's worth adding a link to the python changelog at https://docs.python.org/3/whatsnew/3.8.html, and putting this note under `enhancements` (but that's debatable, up to you)

Reply:
Updated 👍